BibBuddy/app/src/main/java/de/bibbuddy/BookDao.java

```diff
@@ -485,4 +485,63 @@ public Long findShelfIdByBook(Long id) {
     return shelfId;
   }
 
+  /**
+   * Method that finds the shelf name of a book in the database.
```

Comment:
```suggestion
   * Find the shelf name of a book in the database.
```
Comments should describe what they do actively (that is how it is usually done - you might want to check other classes (outside of our project) as examples.